frontend/Dockerfile.playwright

```diff
@@ -4,9 +4,10 @@ ARG PLAYWRIGHT_VERSION
 
 FROM mcr.microsoft.com/playwright:v${PLAYWRIGHT_VERSION}-jammy
 
-ARG PACKAGE_MANAGER
-
```

Comment:
If the arg is no longer used, it can be removed from a number of other places.

https://github.com/WordPress/openverse/blob/25a80213ebc13e8284668d8d86e53a0dee129103/frontend/docker-compose.playwright.yml#L7

https://github.com/WordPress/openverse/blob/25a80213ebc13e8284668d8d86e53a0dee129103/frontend/bin/playwright.sh#L12-L13

Reply:
Removed in [6f3665e](https://github.com/WordPress/openverse/pull/4628/commits/6f3665eff3ee02ca17c41517abebd5c2a57e91ce)